include/libtorrent/torrent.hpp

```diff
@@ -956,6 +956,7 @@ namespace libtorrent
 
 		// LOGGING
 #ifndef TORRENT_DISABLE_LOGGING
+		virtual bool should_log() const override;
```

Comment:
it's not obvious to me why this is virtual. which interface does this implement? Is it something tracker_manager uses?